### ISSUE TITLE
Philosophy section is missing

### ISSUE THREAD

**Issue by pepsiman** [JonasToth/CppCoreGuidelinesTooling#2]
I'd like to enforce P.2

**Comment by JonasToth** [comment]
Yes, I didn't consider Philosophy, I will create the files and mention for P.2 that you are working on it.

Out of curiosity, how would you do that? Explicitly match on everything clang supports and warn or is there already a diagnostic? And should `-pedantic -std=c++11` already do the trick?

**Comment by pepsiman** [comment]
Require -pedantic, -std=c++11 or later.
Don't allow -std=gnu*.
Don't allow non-C++ languages?

**Comment by JonasToth** [comment]
Yes that sounds reasonable.

**Comment by JonasToth** [comment]
I can add you as member of the project if you want.

**Comment by JonasToth** [comment]
Philosophy landed in master.
